Terminal.Gui/Views/Menu/MenuBar.cs

```diff
@@ -1594,11 +1594,9 @@ private MenuBar GetMouseGrabViewInstance (View view)
 
 
     /// <inheritdoc />
-    public bool LoadDemoData (object ctx = null)
+    public bool LoadDemoData<T> (T context)
```

Comment:
These have to include the where clause or they aren't actually implementations of the interface method.

Reply:
Only for the generic method, that is.

The reason it didn't warn you about that is because there is a default implementation, so the class is still valid, but now the default interface implementation is a duplicate on the class that can still be called via the interface like: `IDesignable.LoadDemoData<string>("")` for example.